backend/tracim_backend/views/frontend.py

```diff
@@ -17,15 +18,29 @@
 
 
 class FrontendController(Controller):
-    def __init__(self, dist_folder_path: str):
+    def __init__(self, dist_folder_path: str, custom_toolbox_folder_path: typing.Optional[str]):
         self.dist_folder_path = dist_folder_path
+        self.custom_toolbox_folder_path = custom_toolbox_folder_path
+        self.custom_toolbox_files = []  # typing.List["os.DirEntry"]
```

Comment:
```suggestion
        self.custom_toolbox_files = []  # type: typing.List["os.DirEntry"]
```

```suggestion
        self.custom_toolbox_files = []  # type: typing.List[os.DirEntry]
```

Reply:
same answers as latest : this fix doesn't work on python3.5.